### PR TITLE
chore: clean up readmes

### PR DESCRIPTION
**Pull request by b-admike**
### Description

Fixes https://github.com/strongloop/loopback-next/issues/562

### Checklist

- [ ] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)

### CODE REVIEW

File: packages/authentication/README.md [hunk 1]
```diff
@@ -176,6 +178,15 @@ class MyApp extends Application {
 }
 ```
 
+You can try your authentication component by using your favourite REST client
+and by setting the `authorization` header. Here is an example of what your
+request might look like using cURL:
```
Comment (jannyHou) [inline packages/authentication/README.md]: `cURL` --> `curl` ?

File: README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ LoopBack makes it easy to build modern applications that require complex integra
 - Define your data and endpoints with OpenAPI
 - No maintenance of generated code
 
-# Status: ALPHA
+# Status: BETA
```
Comment (kjdelisle) [inline README.md]: This should not change yet. We're not going to beta just yet, beta implies that the API has solidified, and that we're running up to release in a relatively short time frame.

File: packages/rest/README.md [hunk 3]
```diff
@@ -2,10 +2,9 @@
 
 The REST API package for [loopback-next](https://github.com/strongloop/loopback-next).
 
-## STATUS: ALPHA
-This package is not yet ready for production use, and the API has not yet
-stabilized. Additionally, this document is also a work-in-progress and some
-sections may not have corresponding code!
+## STATUS: BETA
```
Comment (kjdelisle) [inline packages/rest/README.md]: Same as above

File: packages/context/README.md [hunk 4]
```diff
@@ -28,8 +28,7 @@ app.bind('defaultName').to('John'); // setting context
 
 class HelloController {
   // consider this.ctx here
-  constructor(@inject('defaultName') name) { // injecting dependency using context
-    this.name = name;
+  constructor(@inject('defaultName') private name) { // injecting dependency using context
```
Comment (kjdelisle) [inline packages/context/README.md]: I don't think this is valid in either TypeScript or JavaScript. Can we fix this to either have no property access modifiers, or to be valid TypeScript instead? (`@inject('defaultName') private name: string`)

File: packages/repository/README.md [hunk 5]
```diff
@@ -291,15 +311,14 @@ import {
...
-  Filter,
   EntityCrudRepository,
 } from '@loopback/repository';
 
 // The Controller for Note
 class NoteController {
   constructor(
     // Use constructor dependency injection to set up the repository
-    @repository('noteRepo')
+    @repository('repositories.noteRepo')
```
Comment (kjdelisle) [inline packages/repository/README.md]: It's my understanding that the `@repository` sugar methods perform this step for you. The new example would have an injection key of `repositories.repositories.noteRepo` if done this way.

File: packages/repository/README.md [hunk 6]
```diff
@@ -319,7 +338,7 @@ Alternatively, the controller can be declared using property injection:
 
 ```ts
 class NoteController {
-  @repository('noteRepo')
+  @repository('repositories.noteRepo')
```
Comment (kjdelisle) [inline packages/repository/README.md]: Same comment as above.

File: packages/repository/README.md [hunk 7]
```diff
@@ -28,26 +28,37 @@ models. For example,
...
 A repository can be created directly using `DefaultCrudRepository`.
 
 ```ts
+import {DefaultCrudRepository} from '@loopback/repository';
+// also import Note and ds from wherever you defined them
+
   const repo = new DefaultCrudRepository(Note, ds);
 
-  // Bind the repository instance
+  // Bind the repository instance in a context
```
Comment (kjdelisle) [inline packages/repository/README.md]: `// Bind the repository instance to the 'ctx' Context.`

File: packages/repository/README.md [hunk 8]
```diff
@@ -1,6 +1,6 @@
 # @loopback/repository
 
-It provides a common set of interfaces for interacting with databases.
+This module provides a common set of interfaces for interacting with databases.
```
Comment (kjdelisle) [inline packages/repository/README.md]: :+1:

File: packages/repository/README.md (hunk 5 above)
Comment (b-admike) [reply packages/repository/README.md]: That is correct. Good catch!

**Pull request merged**